### ISSUE TITLE
Spoof Accept Language Doesn't Change

### ISSUE THREAD

**Issue by Geyup** [dillbyrne/random-agent-spoofer#439]
Using Firefox 45.2.0 ESR in Linux if I have the ```Spoof Accept Language``` checked and ir I put it on any language it always stays the same.

Isn't this suppose to be the ```intl.accept_languages``` setting?

```extensions.agentSpoof.whiteListAcceptLanguage``` is the only other language setting I have in about:config and they both stay as;

en-US,en;q=0.5
en-US,en;q=0.8

If I uncheck ```Spoof Accept Language``` then ```intl.accept_languages``` changes to ```en-US, en```, this is the only change I see happening.

**Comment by dillbyrne** [comment]
It does change but for now only the firefox profiles have other accept language header optons defined because I had that information available from the downloading the browser in other languages and checking the header. There is no such option for chrome and I think IE and edge may be defined by the OS ?.

As for ESR well that profile is a mirror of the tor browser profile as some users requested it and that always displays a en-us header so that is why there is only one for that.

If a profile has your selected accept language header it will use it but if not it falls back to the en-us variant for that profile since each profile is guaranteed to have at least an en-us accept language header.

Look at the difference between the firefox profiles and the rest here https://github.com/dillbyrne/random-agent-spoofer/blob/master/data/json/useragents.json

If you can find reliable information about the accept language headers for the other browsers I can add them. IE google chrome's default accept language header for France etc

**Comment by Geyup** [comment]
I'm talking about Firefox here, I'm not sure where you get this information from that ESR is a mirror of Tor, more like the other way around, Tor is a mirror of ESR...

Anyhow ESR is Firefox, so I'm not understanding why this should not work, unless because you're saying users requested this, and that is only the reason why? Then I'm not understanding that either, as to why others wouldn't want to spoof it, because then, was is the poiint of having an option, which makes the end-user think this is good coming from the developer and then end-users decided what they like...

Hmm

**Comment by Geyup** [comment]
Can I please get some clarification on my last reply?

thank you

**Comment by dillbyrne** [comment]
@Geyup Sure Sorry. 

To clarify the RAS ESR profile is uses the UA for the tor browser. Users were asking for a tor browser profile but I did not want to call the profile Tor since some users would equate that profile with the tor browser. At the time I added the ESR profile as US english only since it would be more common on the internet.  You are right though, It makes sense for the ESR profile to have all the other languages as well.
